brands/office/insurance.json

```diff
@@ -0,0 +1,90 @@
+{
+  "office/insurance|Allstate": {
+    "countryCodes": ["us"],
+    "matchNames": ["allstate insurance"],
+    "tags": {
+      "brand": "Allstate",
+      "brand:wikidata": "Q2645636",
+      "brand:wikipedia": "en:Allstate",
+      "name": "Allstate",
+      "office": "insurance"
+    }
+  },
+  "office/insurance|American Family Insurance": {
+    "countryCodes": ["us"],
+    "matchNames": ["american family", "amfam"],
+    "tags": {
+      "brand": "Allstate",
```

Comment:
should be "American Family Insurance"  (this probably copy-pasted from "Allstate"?)